src/main/java/com/kodcu/service/MarkdownService.java

```diff
@@ -21,7 +21,7 @@
 @Component
 public class MarkdownService {
 
-    private final Logger logger = LoggerFactory.getLogger(MarkdownService.class);
+    private static final Logger logger = LoggerFactory.getLogger(MarkdownService.class);
```

Comment:
What's the motivation of making it static here?

`MarkdownService` is supposed to be used as singleton, so no space savings. OTOH you risk potentially threading issues depending on the Logger implementation.